pretyping/typeclasses.mli

```diff
@@ -83,8 +83,8 @@ val is_instance : global_reference -> bool
 (** Returns the term and type for the given instance of the parameters and fields
    of the type class. *)
 
-val instance_constructor : typeclass Univ.puniverses -> constr list ->
-  constr option * types
+val instance_constructor : typeclass EConstr.puniverses -> EConstr.t list ->
+  EConstr.t option * EConstr.t
```

Comment:
The other functions in this file could also be econstr.

Reply:
I am not sure, it looks to me like a few of these functions are expected to return ground terms, right?

Reply:
When I looked there were a few which did unsafe to constr, I would say at least those should use econstr.